src/main/kotlin/uk/gov/justice/digital/hmpps/pecs/jpc/location/importer/Court.kt

```diff
@@ -1,7 +1,3 @@
 package uk.gov.justice.digital.hmpps.pecs.jpc.location.importer
 
-object Court : LocationFromCells {
-
-    override val sheetIndex = 5
-    override val sheetName = "COURT"
-}
+object Court : LocationFromCells(5, "COURT")
```

Comment:
I'm not sure that these want to be singletons.  Also why not just stick them all into the same file?  Possibly event next to the interface.